generator/svelte/src/lib/utils.ts

```diff
@@ -16,10 +16,16 @@ export const convertToRelativeTime = (isoDate: string) => {
 	return dayjs.utc(isoDate).fromNow();
 };
 
-export const convertToCalendarTime = (isoDate?: string) => {
-	if (!isoDate) return null;
+export const convertToCalendarTime = (dateStr?: string) => {
+	if (!dateStr) return null;
 
+<<<<<<< Updated upstream
```

Comment:
Please resolve conflicts.

Reply:
@hammady Updated, I added a screenshot in the description (after setting current datetime to Nov 26, all seems well)